react/features/dropbox/functions.native.ts

```diff
@@ -28,9 +28,11 @@ export async function _authorizeDropbox(_appKey?: any, _redirectURI?: any): Prom
 /**
  * Gets a new access token based on the refresh token.
  *
+ * @param {string} _appKey - The dropbox appKey.
+ * @param {string} _rToken - The refresh token.
  * @returns {Promise}
  */
-export function getNewAccessToken() {
+export function getNewAccessToken(_appKey: string, _rToken: string) {
```

Comment:
On web. But because the function is called in a common file (used on both web and native) we need the native function to have the same signature as the web one. And this is why they start with '_' (eslint ignores unused vars rule for them)